crates/miden-lib/asm/miden/contracts/auth/basic.masm

```diff
@@ -111,30 +112,25 @@ end
 
 #! Creates the transaction summary and returns it in the order in which it will be hashed.
 #!
-#! Inputs:  [final_nonce]
+#! Inputs:  [SALT]
 #! Outputs: [SALT, OUTPUT_NOTES_COMMITMENT, INPUT_NOTES_COMMITMENT, ACCOUNT_DELTA_COMMITMENT]
 #!
 #! Where:
-#! - SALT is used for replay protection and contains the final nonce of the account and the
-#!   block number of the transaction's reference block.
+#! - SALT is a user provided randomizer.
```

Comment:
not super happy about this, lmk if you guys have another idea. the goal is to generalize across single & multisig procedures (also - if they share `create_tx_summary`, is it ok to leave this proc here?)

Reply:
Following from https://github.com/0xMiden/miden-base/issues/1640#issuecomment-3125913824, I think we should create a new `miden::auth` module and put this and also `hash_tx_summary` procedures there.